### PR TITLE
added files

### PR DESCRIPTION
**Pull request by faikp**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    0%{
+        background-position: 0% 50%;
+    }
+    50%{
+        background-position: 100% 50%;
+    }
+    100%{
+        background-position: 0% 50%;
+    }
+}
```
Comment (Abbas4tech) [inline style.css:51]:
```suggestion
    animation: bg 4s infinite alternate ;
}
@keyframes bg {
    from {
        background-position: 0 0;
    }
    to {
        background-position: 100% 100%;
    }
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    grid-template-columns: repeat(auto-fill, minmax(300px,1fr));
+    grid-template-rows: 100px,auto;
+    grid-auto-flow: dense;
+    gap: 10px;
+}
+.container .box{
+    background: linear-gradient(45deg,#2c3e50,#27ae60,#2980b9,#e74c3c,#8e44ad);
+    padding: 20px;
+    font-size: 20px;
+    background-size: 400% 400%;
```
Comment (Abbas4tech) [inline style.css:32]:
```suggestion
    background-size: 200% 200%;
```

Comment (Abbas4tech) [inline style.css]:
```suggestion
    background-size: 200% 200%;
```

**Review by anask601 — APPROVED** [review]
Amazing work @faikp 😎

File: style.css (hunk 1 above)
Comment (Abbas4tech) [inline style.css]:
```suggestion
    animation: bg 4s linear infinite alternate ;
}
@keyframes bg {
    from {
        background-position: 0% 0%;
    }
    to {
        background-position: 100% 100%;
    }
}
```

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Improved your animation @faikp . added `alternate` for smooth color changing. also changes animation position to apply whole color gradient properly

**Review by Rehan-Khan-Engineer — APPROVED** [review]
Nicely Done @faikp

**Review by syedabdullahali — COMMENTED** [review]
awesome

**Review by syedabdullahali — APPROVED** [review]

**Review by nehalskdev — APPROVED** [review]

**Pull request merged**